File: DServer/src/main/java/com/vovamisjul/dserver/tasks/RunningTaskInfo.java
package com.vovamisjul.dserver.tasks;

public class RunningTaskInfo {

    private final String copyId;
    private final TaskInfo taskInfo;
    private final String authorId;

    public RunningTaskInfo(TaskInfo taskInfo, String copyId, String authorId) {
        this.taskInfo = taskInfo;
        this.copyId = copyId;
        this.authorId = authorId;
    }

    public String getCopyId() {
        return copyId;
    }

    public TaskInfo getTaskInfo() {
        return taskInfo;
    }

    public String getAuthorId() {
        return authorId;
    }
}

File: DistributionWorker/DistributionWorker/Tasks/RunningTaskInfo.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TasksBase;

namespace DistributionWorker.Tasks
{
    public class RunningTaskInfo
    {
        [JsonProperty(PropertyName = "copyId")]
        public string CopyId { get; set; }
        [JsonProperty(PropertyName = "taskInfo")]
        public TaskInfo TaskInfo { get; set; }
        [JsonProperty(PropertyName = "comment")]
        public string Comment { get; set; }
        [JsonProperty(PropertyName = "created")]
        public DateTime Created { get; set; }
        [JsonProperty(PropertyName = "author")]
        public string Author { get; set; }
    }
}
